GitUI/CommandsDialogs/BrowseDialog/FormOpenDirectory.cs

```diff
@@ -166,19 +166,10 @@ private static GitModule OpenGitRepository([NotNull] string path, ILocalReposito
             return chosenModule;
         }
 
-        internal TestAccessor GetTestAccessor()
-            => new TestAccessor(this);
-
         internal readonly struct TestAccessor
         {
-            private readonly FormOpenDirectory _form;
-
-            public TestAccessor(FormOpenDirectory form)
-            {
-                _form = form;
-            }
-
-            public GitModule OpenGitRepository([NotNull] string path, ILocalRepositoryManager localRepositoryManager) => FormOpenDirectory.OpenGitRepository(path, localRepositoryManager);
+            public static GitModule OpenGitRepository([NotNull] string path, ILocalRepositoryManager localRepositoryManager)
+                => FormOpenDirectory.OpenGitRepository(path, localRepositoryManager);
```

Comment:
📝 It's fine to make this static, but the `TestAccessor` pattern is quite specific so I would prefer to keep the constructor and `GetTestAccessor()` methods.

Reply:
I tought about keeping them but decided to remove them because they are not used and there's only a low likelihood for additional tests here.